chalicelib/checks/release_updates_checks.py

```diff
@@ -601,7 +606,8 @@ def data_release_updates(connection, **kwargs):
             'static_section': report_static_section
         }
         check.description = 'Ready to publish new data release updates.' + used_res_strings
-        check.action_message = 'Will publish %s  grouped updates with the update_tag: %s. See brief_output.' % ( str(len(group_reports)), kwargs['update_tag'])
+        check.action_message = 'Will publish %s  grouped updates with the update_tag: %s. See brief_output.' % (
```

Comment:
There are two spaces after `%s` that should probably just be one. This looks like an old problem. Not super important.